osu.Game/Screens/SelectV2/WedgeDifficultyDisplay.cs

```diff
@@ -0,0 +1,365 @@
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                            Padding = new MarginPadding { Bottom = border_weight, Right = border_weight },
+                            Child = new Container
+                            {
+                                RelativeSizeAxes = Axes.X,
+                                AutoSizeAxes = Axes.Y,
+                                Masking = true,
+                                CornerRadius = 10 - border_weight,
+                                Shear = shear,
+                                Children = new Drawable[]
+                                {
+                                    new Box
+                                    {
+                                        RelativeSizeAxes = Axes.Both,
+                                        Colour = colourProvider.Background5,
+                                    },
+                                    new GridContainer
+                                    {
+                                        RelativeSizeAxes = Axes.X,
+                                        AutoSizeAxes = Axes.Y,
+                                        Padding = new MarginPadding { Left = SongSelect.WEDGE_CONTENT_MARGIN, Right = 20f, Top = 7.5f, Bottom = 5f },
+                                        Shear = -shear,
+                                        RowDimensions = new[] { new Dimension(GridSizeMode.AutoSize) },
+                                        ColumnDimensions = new[]
+                                        {
+                                            new Dimension(),
+                                            new Dimension(GridSizeMode.AutoSize),
+                                        },
+                                        Content = new[]
+                                        {
+                                            new Drawable[]
+                                            {
+                                                hitStatisticsDisplay = new WedgeHitStatisticsDisplay
+                                                {
+                                                    RelativeSizeAxes = Axes.X,
+                                                },
+                                                difficultyStatisticsFlow = new DifficultyStatisticsFlow
+                                                {
+                                                    AutoSizeAxes = Axes.Both,
+                                                    Spacing = new Vector2(8f, 0f),
+                                                    Padding = new MarginPadding { Left = 10f },
+                                                    Children = new[]
+                                                    {
+                                                        firstStatisticDifficulty = new WedgeStatisticDifficulty(BeatmapsetsStrings.ShowStatsCs) { Width = 65 },
```

Comment:
The fixed width doesn't work for other languages. The statistic can probably be autosized x, and the bar fixed to 65. I think it looks fine if the text is exceeds the bar (in english, the approach rate text is 70 and exceeds it).

Reply:
I've adjusted that in some form in https://github.com/ppy/osu/pull/32480/commits/6ca36855e46459d0f06a29fc144465eb0d883f41, now all statistics accumulate a width enough for the longest label across them.

Reply:
Size invalidation doesn't seem to work reliably. Can reproduce when pressing up/down arrows on language dropdown.
![Screenshot 2025-03-27 at 12 41 20 PM](https://github.com/user-attachments/assets/132b6c65-1358-4512-b797-6a419d1c02a7)

Also I'm not really fond of this extreme case (one long translation and all others are wide):
![Screenshot 2025-03-27 at 12 37 22 PM](https://github.com/user-attachments/assets/ee17089a-667e-4f8e-93f9-8fc534a15ae8)

Reply:
> Also I'm not really fond of this extreme case (one long translation and all others are wide):

I'm not sure that can be improved besides adding some spacing between the count and difficulty statistics, unless you have any bright ideas I'm not interested in dissecting this.

Reply:
As I've said in the initial review, have the bar have a fixed width and text autosize. I consider this less broken than the huge amounts of whitespace in my previous image:
![Screenshot 2025-03-27 at 8 47 20 PM](https://github.com/user-attachments/assets/c5245782-ce49-4e19-a2d0-ec62e381b248)

The figma also shows approach rate text width slightly over than the bar, unless that's an oversight.

Reply:
How would this look if a statistic in the middle has a longer label than everything else, the bars will no longer be equally spaced and it'll look awkward in my eyes.

Reply:
You might have a point, so I'll just wait for other opinions.

![Screenshot 2025-03-27 at 9 16 54 PM](https://github.com/user-attachments/assets/ccbbd206-c328-4f3e-ab2d-be719eaa2e89)